documentation/src/main/asciidoc/stories/assembly_server_reference.adoc

```diff
@@ -1,14 +1,10 @@
 [id='server_reference']
-:context: server
+:context: server-info
 = Reference
 
-//Community
```

Comment:
@oraNod is this intentional? (removing from community)

Reply:
ps. I think this file isn't consumed anywhere now

Reply:
@pruivo Yes, this is not included in the community edition of the Server Guide. I do want to leave the assembly in the community doc set. I'm planning to come back to this and figure it out separately from this PR.